tests/FSharpVSPowerTools.Tests/FSharpVSPowerTools.Tests.fsproj

```diff
@@ -288,6 +288,22 @@
     </When>
   </Choose>
   <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v4.5' Or $(TargetFrameworkVersion) == 'v4.5.1' Or $(TargetFrameworkVersion) == 'v4.5.2' Or $(TargetFrameworkVersion) == 'v4.5.3' Or $(TargetFrameworkVersion) == 'v4.6' Or $(TargetFrameworkVersion) == 'v4.6.1')">
+      <ItemGroup>
+        <Reference Include="FSharp.Compiler.Service.ProjectCracker">
+          <HintPath>..\..\packages\test\FSharp.Compiler.Service.ProjectCracker\lib\net45\FSharp.Compiler.Service.ProjectCracker.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+        <Reference Include="FSharp.Compiler.Service.ProjectCrackerTool">
```

Comment:
I'm not sure you should be referencing this.  The [unit tests DLL for FCS.ProjectCracker](https://github.com/fsharp/FSharp.Compiler.Service/blob/master/tests/service/FSharp.Compiler.Service.Tests.fsproj#L89) doesn't reference ProjectCrackerTool.  Has paket added the reference automatically?